nbs/12_cli.ipynb

```diff
@@ -61,6 +61,14 @@
     "> CLI commands"
```

Comment:
<p>Won't removing the hyphens mean we end up with two headings instead of one? (Since nbdev auto-adds a heading for the function itself)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/866/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I did this because `FilterDefaults` is eating up a whole bunch of functions:

<img width="193" alt="image" src="https://user-images.githubusercontent.com/559360/184853536-30d4f87f-076a-4593-aae5-b674ad597688.png">

Maybe a better solution would be if we could control the section level of `FilterDefaults`? Or make it internal (but maybe manually add it to `__all__`)? What do you think?

Reply:
To move this PR along, I'll put `FilterDefaults` and `nbdev_filter` into a visible `Filter` section for now, and make all the other sections hidden again

Reply:
Ok, I went with `exporti` + `_all_` + `show_doc(..., title_level=4)` for now (and fixed #869 to make that work)